lib/internal/bootstrap/realm.js

```diff
@@ -320,14 +320,17 @@ class BuiltinModule {
     );
   }
 
-  static getCanBeRequiredByUsersWithoutSchemeList() {
-    return ArrayFrom(canBeRequiredByUsersWithoutSchemeList);
-  }
-
   static getSchemeOnlyModuleNames() {
     return ArrayFrom(schemelessBlockList);
   }
 
+  static getAllBuiltinModuleIds() {
+    return [
+      ...canBeRequiredByUsersWithoutSchemeList,
+      ...ArrayFrom(schemelessBlockList, (x) => `node:${x}`),
+    ];
```

Comment:
https://github.com/nodejs/node/blob/c42d8461b061136da2b3c04bd976d2e15b81969d/doc/contributing/primordials.md#L262-L270
```suggestion
    const allBuiltins = ArrayFrom(canBeRequiredByUsersWithoutSchemeList);
    ArrayPrototypePushApply(allBuiltins, ArrayFrom(schemelessBlockList, (x) => `node:${x}`));
    return allBuiltins;
```

Reply:
This is during bootstrapping, so it should not have an impact. I rather keep the current code.

Reply:
We should add a commit, it's not obvious that it's running during bootstrapping IMO